content/en/docs/userguide/try_waypoint.md

```diff
@@ -0,0 +1,169 @@
+bookinfo-reviews-istio-waypoint-5d544b6d54-v5tc9   1/1     Running   0          4s
+details-v1-5f4d584748-bz42z                        1/1     Running   0          4m35s
+productpage-v1-564d4686f-2rjqc                     1/1     Running   0          4m35s
+ratings-v1-686ccfb5d8-dnzkf                        1/1     Running   0          4m35s
+reviews-v1-86896b7648-fqm4z                        1/1     Running   0          4m35s
+reviews-v2-b7dcd98fb-nn42q                         1/1     Running   0          4m35s
+reviews-v3-5c5cc7b6d-q4r5h                         1/1     Running   0          4m35s
+sleep-9454cc476-86vgb                              1/1     Running   0          4m25s
+```
+  
+Replace the waypoint image with the Kmesh customized image. Based on istio-proxy, Kmesh adds an customized listener filter called [Kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv), which will parse the custom TLV protocol encoded by Kmesh and obtain the target address and metadata to connect L4 and L7.
+
+```bash
+[root@ ~]# kubectl get gateways.gateway.networking.k8s.io
+NAME               CLASS            ADDRESS         PROGRAMMED   AGE
+bookinfo-reviews   istio-waypoint   10.96.207.125   True         8m36s
+```
+
+Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0" to the `bookinfo-reviews` gateway, convert `{arch}` to the architecture of the host, current optional values are `x86` and `arm`. Then gateway pod will restart. Now Kmesh is L7 enabled!
+
+### Apply weight-based routing
+
+Configure traffic routing to send 90% of requests to `reviews` v1 and 10% to `reviews` v2
+
+```bash
+[root@ ~]# kubectl apply -f -<<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+    - reviews
+  http:
+  - route:
+    - destination:
+        host: reviews
+        subset: v1
+      weight: 90
+    - destination:
+        host: reviews
+        subset: v2
+      weight: 10
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+  - name: v3
+    labels:
+      version: v3
+EOF
+```
+
+Confirm that roughly 90% of the traffic go to `reviews` v1
+
+```bash
+[root@ ~]# kubectl exec deploy/sleep -- sh -c "for i in \$(seq 1 100); do curl -s http:productpage:9080/productpage | grep reviews-v.-; done"
+```
+
+### Understanding what happend
+
+Because `default` namespace has been managed by Kmesh and we have deployed a waypoint proxy for service account `bookinfo-reviews`, so all traffic sent to service `reviews` will be forwarded to waypoint by Kmesh. Waypoint will send 90% of requests to `reviews` v1 and 10% to `reviews` v2 according to the route rules we set.
```

Comment:
```suggestion
Because `default` namespace has been managed by Kmesh and we have deployed a waypoint proxy for service account `bookinfo-reviews`, so all traffic sent to service `reviews` will be forwarded to waypoint by Kmesh. Waypoint will send 90% of requests to `reviews v1` and 10% to `reviews v2` according to the route rules we set.
```